ACMAS/app/ACMAS_Web/search.py

```diff
@@ -117,6 +119,8 @@ def getFiles(self, uni, course, fType):
         # If course is specified
         if course is not None:
             # If course is in University, return file QuerySet, otherwise return None
+            # if courses.filter(Name=course).exists():
+            #     return self.fileSearch.getFiles(courses.get(Name=course), fType)
```

Comment:
Throughout--remove all this dead code if we don't need it.

Reply:
Resolved